server/routes/employer-sites.js

```diff
@@ -77,7 +78,31 @@ router.patch(
   })
 );
 
-// Read: Get All
+// Read: Get *All* Sites
+router.get(
+  '/all',
```

Comment:
What do you think of making this endpoint

`GET /sites`

and the other endpoint

`GET /user/sites`?

Reply:
While I feel lazy about it, I do agree that it makes sense. I'll get to it.

Reply:
Feel okay about

`GET /` to get all

`GET /user` to get user specific?